e2e/testcases/helm_sync_test.go

```diff
@@ -387,32 +389,35 @@ func TestHelmLatestVersion(t *testing.T) {
 
 	// helm-sync automatically detects and updates to the new helm chart version
 	newVersion = "2.5.9"
-	chart, err = nt.BuildAndPushHelmPackage(nomostest.RootSyncNN(configsync.RootSyncName),
+	chart, err = nt.BuildAndPushHelmPackage(rootSyncID.ObjectKey,
```

Comment:
Should `nt.BuildAndPushHelmPackage` just update the ChartID, so that it doesn't need to be manually updated below?

Reply:
Could do that. Tho realistically the NN passed to BuildAndPushHelmPackage is only used for temp file location and artifacts path. It might make more sense to move those to be based on the helm chart name instead, so more than one RSync can use the same chart, if needed.

Updating the expectation for helm charts could use a helper function though, to make it less verbose.

Reply:
Think I'm gonna save that optimization for a follow up PR.

Reply:
unresolving to keep a record of follow up PRs TODO.

Reply:
Moving this into the Helm RSync constructors as part of https://github.com/GoogleContainerTools/kpt-config-sync/pull/1406